shuup/importer/transforms.py

```diff
@@ -67,7 +68,7 @@ def process_data(rows):
     if not len(data):
         for y, row in enumerate(rows):
             if y == 0:
-                headers = [x.lower().strip() for x in row]
+                headers = [force_text(x).lower().strip() for x in row]
```

Comment:
Who's passing non-unicode data to this function anyway? IMHO it's the callers responsibility to make sure that the passed-in data is already decoded. So rather add a docstring which documents that values in rows should be text, not bytes, and then check the callers that they do proper decoding.